CatProducer/python/catSetup_cff.py

```diff
@@ -39,8 +39,11 @@ def catSetup(process, runOnMC=True, doSecVertex=True):
         ## Load MC dependent producers
         process.load("CATTools.CatProducer.pdfWeight_cff")
         process.load("CATTools.CatProducer.pileupWeight_cff")
+        process.load("CATTools.CatProducer.pseudoTop_cfi")
         process.out.outputCommands.append("keep *_pdfWeight_*_*")
         process.out.outputCommands.append("keep *_pileupWeight_*_*")
+        process.out.outputCommands.append("keep *_pseudoTop_*_*")
+        process.out.outputCommands.append("keep *_partonTop_*_*")
```

Comment:
Can we maintain the output in catEventContent__cff single file? so we don't need to look around here and there.